### DISCUSSION TITLE
irreparable damage

### DISCUSSION THREAD

**Discussion by v74863** [ZerBea/hcxdumptool#430] [General]
> This is a highly experimental penetration testing tool!
> It is made to detect vulnerabilities in your NETWORK mercilessly!
> Misuse within a network, without specific authorization, may cause
> irreparable damage and result in significant consequences!

Hi, Mike
Please clarify what you mean by **irreparable damage**.
The worst-case scenarios.
Is it possible to burn the electronics?
Thanks in advance.

**Comment by ZerBea** [comment]
hcxdumptool can't cause direct harm - but collateral damage (as mentioned below) is possible.

It can cause a full reset of the target. This includes deleting the default configuration (of the NVRAM) inclusive the default PSK (if generated by factory). This values are gone forever! That is a very rare case, but it is possible!

It can break an active internet connection between an IoT (sensor, cam, motion control) and a server.
If that's important data ,e,g, needed by heat control it can cause irreparable damage.

**Reply by sudo-nano** [reply]
Sorry if this is a noob question, but how would `hcxdumptool` cause a full reset?

**Comment by ZerBea** [comment]
That can happen if you target a router which is running a buggy firmware:
https://github.com/ZerBea/hcxdumptool/issues/246#issue-1533775961
